modules/core/src/v2/coins/eth.ts

```diff
@@ -895,17 +1198,18 @@ export class Eth extends BaseCoin {
    * @returns void
    * @throws Error if The prebuild is invalid
    */
-  validateHopPrebuild(wallet: any, hopPrebuild: HopPrebuild, originalParams: any, callback: any): void {
-    return co(function *() {
-      const {
-        tx,
-        id,
-        signature,
-      } = hopPrebuild;
-
+  validateHopPrebuild(
+    wallet: Wallet,
```

Comment:
hmm it doesn't look like ```wallet``` is used in this function?

Reply:
Yep,  you're right, it is definitely unused. I'll clean it up in the next PR which is almost ready.